### PR TITLE
Responsive Design, Project 1 - Design of a Responsive clone of The Next Web home page

### PR DESCRIPTION
**Pull request by gango-anan**
## In this project we : 
- created the page structure in the index.html file using semantic html elements and laid them out according to the lay out on the original page.
- then styled the page to position the elements in there respective locations based on the original page.
- worked on the typography of the entire document.
- added style to the page for the mobile version.
- added style to the page for the table and desktop versions using media queries.

### CODE REVIEW

**Review by layzrd — CHANGES_REQUESTED** [review]
### Status: Require changes 🌳 ♲
Hi @gango-anan , 👏 You did very well on this project. Most requirements have done 👌, Kindly fix the following issues.

[REQUIRED]

- [ ] On medium screens, the TNW logo must appear in the top left of the page.
- [ ] For Latest Deals and Latest news
        -  On small screens, each article should have filled a full row.
- [ ] No small breakpoint as it is mobile-first and the base styles are for the mobile screen. (Screenshot on [mobile-first approach](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/responsive_design/mobile_first.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540217/))
 	_medium: min-width around 768px
 	large: min-width 1024px_
- [ ] Do not repeat CSS rules; set a separate class name for repetitive CSS rules.

Once you fix the issues, kindly submit another code review ♲
 Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/)

**Comment by gango-anan** [comment]
Powerful feedback, thank you.

**Comment by gango-anan** [comment]
Raised issues from the previous code review have been fixed and a request for a new review submitted awaiting approval, thanks.

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Change Required :recycle: 
## Hi @gango-anan 

- You've done a great job so far and that's very commendable. 
- Good job implementing the change as requested by the previous TSE.

However, there's still a change you need to effect before your project can be approved.

## [REQUIRED]

- [ ] On small screen and medium screen, the text on the latest news section are too tiny, same goes for the latest funding section.

Kindly submit for another review when you're done making the change.

## Happy Coding :computer: 

_Feel free to reach out to me on slack_ **`@Elijah Ayandokun`**
  **Elijah O. Ayandokun**
[Github](https://github.com/ElijahTobs) | [LinkedIn](https://www.linkedin.com/in/ayandokunelijah/) | [Twitter](https://twitter.com/ElijahTobs)

**Comment by gango-anan** [comment]
Thanks for the great observations, all issues fixed and recommended changes put in place, a new code review requested awaiting approval

File: README.md
Comment (descholar-ceo) [inline README.md]: This link is not right and it's giving a 404, please change it.  It should be [this one](https://github.com/gango-anan/the-next-web-clone/issues)

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @gango-anan and @shahnaeem  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes]
- [ ] The next web logo in navbar and images in the latest news esction are strecthed on mobile screens, Advice: you can use fixed heights in your media queries for specific screen sizes

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Comment by gango-anan** [comment]
Noted, thanks

**Comment by gango-anan** [comment]
Changes done, submitted and submitted for another review...

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @gango-anan! 👋 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:


### You may merge, good luck with your next projects! :rocket:
### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**
